app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HelloWorld</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
- [ ] Nice job with the API endpoint. It returns a different greeting every time you call it.

![image](https://user-images.githubusercontent.com/68796232/185271560-4900fe12-53ab-4d64-86a5-ef0b7a6731e6.png)

- [ ] [Changes Required] When I run the rails server and open the root route it through the error in the image below, So please fix that and ensure that the application is running as expected.

![image](https://user-images.githubusercontent.com/68796232/185271780-c57647e4-2e31-4ab3-a726-70ca2fb8d13a.png)

Reply:
That's odd, it runs fine from my end...not sure what to change :/

Reply:
* Ensure you have all setup instructions on your readme file , I think why the CR is getting errors is because dependencies have not been installed for `jsbundling-rails` `react-dom` `babel-loader` . since you are running the app locally you wont have an issue but for someone who cloned your app and install has to do run `yarn install` to install all dependencies needed for the app to work. you can find the dependencies in the `package.json file`